### PR TITLE
Initial kinesis-to-share example

### PR DESCRIPTION
**Pull request by brian-vendia**

### CODE REVIEW

File: share/kinesis-to-share/README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+* [AWS Serverless Application Model CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+
+* [Docker](https://docs.docker.com/install/)
+
+* [Vendia Share CLI](https://vendia.net/docs/share/cli)
+
+## Installing Python3 Dependencies
```
Comment (cory-vendia) [inline share/kinesis-to-share/README.md:27]: Should this be after the instructions to clone the repo and switch working directories?

File: share/kinesis-to-share/README.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+```bash
+share login
+```
+
+The `share uni create` command can be used to deploy our Uni.  You will need to copy the file `registration.json.sample` to `registration.json`.  Pick a unique `name` for your uni that begins with `test-` - by default all Unis share a common namespace so here is your chance to get creative.  Update the `userId` attribute of each node to reflect your Vendia Share email address.
+
+```bash
+cd uni_configuration
+share uni create --config registration.json
```
Comment (cory-vendia) [inline share/kinesis-to-share/README.md:72]: should point to uni_configuration/registration.json given the current path

**Review by cory-vendia — CHANGES_REQUESTED** [review]
Works well. Just a few changes to the README needed.

File: share/kinesis-to-share/README.md (hunk 2 above)
Comment (brian-vendia) [reply share/kinesis-to-share/README.md:72]: The `share` command should be run from the `uni_configuration` directory (previous line).

File: share/kinesis-to-share/README.md (hunk 1 above)
Comment (brian-vendia) [reply share/kinesis-to-share/README.md]: Good call - made the change.

**Pull request merged**